Kernel/Devices/HID/PS2/KeyboardDevice.cpp

```diff
@@ -47,7 +47,23 @@ UNMAP_AFTER_INIT ErrorOr<NonnullOwnPtr<PS2KeyboardDevice>> PS2KeyboardDevice::tr
 
 UNMAP_AFTER_INIT ErrorOr<void> PS2KeyboardDevice::initialize()
```

Comment:
Commit title should be "Kernel: Detect PS2 keyboards on some chromebooks properly"

Reply:
Done